lib/octokit/client/source_import.rb

```diff
@@ -9,9 +9,9 @@ module SourceImport
       # Start a source import to a GitHub repository using GitHub Importer.
       #
       # @param repo [Integer, String, Hash, Repository] A GitHub repository.
-      # @param vcs [String] The originating VCS type. Can be one of "subversion", "git", "mercurial", or "tfvc".
       # @param vcs_url [String] The URL of the originating repository.
       # @param options [Hash]
+      # @option options [String] :vcs The originating VCS type. Can be one of "subversion", "git", "mercurial", or "tfvc".
```

Comment:
@joeyw I really like the idea of gracefully deprecating this and that looks like a great way to do it. I'll play around with the method you provided and see if I can think of a way to cover the case when no options are passed - for example:

`@client.start_source_import(@repo.full_name, "svn", "https://rphotogallery.googlecode.com/svn/")`